template.go

```diff
@@ -0,0 +1,32 @@
+package cloudconfig
+
+import (
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/k8scloudconfig/v_0_1_0"
+)
+
+type version string
```

Comment:
This should be exported. `NewTemplate` is exported.

Reply:
Oh, I got it. you don't want to allow creating custom ones outside the package. Clever!

Reply:
Not sure godoc-wise though. I leave it up to you :D